homeassistant/components/launch_library/diagnostics.py

```diff
@@ -21,7 +21,9 @@ async def async_get_config_entry_diagnostics(
         return {}
     next_launch = coordinator.data["upcoming_launches"][0]
     starship_launch = coordinator.data["starship_events"].upcoming.launches[0]
+    starship_event = coordinator.data["starship_events"].upcoming.events[0]
```

Comment:
This needs a guard, currently, there are no active events
The other 2 also need a guard, so maybe something generic?

```
def _first_element(data: list[Launch | Event) -> Launch | Event | None:
	if not data:
		return None
	return data[0].raw_data_contents
```

Reply:
Will add this. 

What is the diagnostics thing for? Just so i have a better clue as to what it does 😃

Reply:
That sounds awesome! Thanks 😃